pkg/generator/helper/helper.go

```diff
@@ -148,7 +158,9 @@ func {{.CobraCmdFuncName}}() *cobra.Command {
 			handler := func() {
 				config.RemoveVaultSecretFiles()
```

Comment:
`.persist` needs to run before deletion of secrets

Reply:
Currently `reports.persist` is still called afterwards.

Reply:
I misunderstood this commit https://github.com/SAP/jenkins-library/pull/3034/commits/1ae66d20c1b982897f7c7e627a0c8b00feabde0a and reverted the changes. I'll fix it now